cmd/statusd/whispercfg.go

```diff
@@ -13,22 +13,10 @@ import (
 func whisperConfig(nodeConfig *params.NodeConfig) (*params.NodeConfig, error) {
 	whisperConfig := nodeConfig.WhisperConfig
 	whisperConfig.Enabled = true
-	whisperConfig.IdentityFile = *identityFile
-	whisperConfig.EnablePushNotification = *enablePN
 	whisperConfig.EnableMailServer = *enableMailServer
 	whisperConfig.MinimumPoW = *minPow
 	whisperConfig.TTL = *ttl
 
-	if whisperConfig.EnablePushNotification && whisperConfig.IdentityFile == "" {
-		return nil, errors.New("notification server requires -identity file to be specified")
-	}
-
-	if whisperConfig.IdentityFile != "" {
```

Comment:
`IdentityFile` may be useful to make sure the nodeID stays the same.

Reply:
Scratch that. Let's leave it removed. If we hit some issues it should be implemented from scratch as this should not be a part of Whisper config if used as a nodeID.